src/components/Teams.tsx

```diff
@@ -0,0 +1,75 @@
+import { Backdrop, Box, Stack } from "@mui/material";
+import React, { useState } from "react";
+import { Person as PersonType, Team as TeamType } from "../types";
+import { AutoColumnOnSmall } from "../utils/auto";
+import { Person } from "./Person";
+import { SiteTheme } from "../theme";
+import { People } from "../data/people";
+import { Teams } from "../data/teams";
+
+type TeamMemberProps = {
+  haveSupplementalInfo: boolean;
+  teamName: string;
+  person: PersonType;
+  supplementalInfo: string | null;
+};
+
+export const TeamMember = ({ haveSupplementalInfo, teamName, person, supplementalInfo }: TeamMemberProps) => {
+  const [showOverlay, setShowOverlay] = useState(false);
+
+  return (
+    <Stack
+      direction={AutoColumnOnSmall("row", "column", "sm")}
+      key={`Team-${teamName}-GridItem-${person.names.first}${person.names.last}`}
+    >
+      <Backdrop
+        onClick={() => setShowOverlay(false)}
+        open={showOverlay}
+        sx={{ p: 2, zIndex: SiteTheme.zIndex.tooltip }}
+      >
+        <Person embeddedIn={teamName} isBadge={false} person={person} sx={{ minWidth: 240, padding: 4 }} />
+      </Backdrop>
+      <Person
+        embeddedIn={teamName}
+        isBadge
+        onAvatarClick={() => setShowOverlay(true)}
+        person={person}
+        sx={{ minWidth: 240 }}
+      />
+      {haveSupplementalInfo && <Box sx={{ fontWeight: "bold", lineHeight: "40px", marginLeft: "6px" }}>{supplementalInfo}</Box>}
+    </Stack>
+  );
+};
+
+export const Team = ({ team }: { team: TeamType }) => {
+  const haveSupplementalInfo = !!team.members.find((el) => !!el[1]);
+
+  return (
+    <Stack className="card" gap={2} p={2}>
+      <h2 style={{ margin: 0 }}>{team.name}</h2>
+      <h4 style={{ fontWeight: "normal", margin: 0, marginBlockEnd: 0 }}>{team.description}</h4>
+      {team.members.map(([name, supplementalInfo]) => (
+        <TeamMember
+          haveSupplementalInfo={haveSupplementalInfo}
+          teamName={team.name}
+          person={People[name]}
+          supplementalInfo={supplementalInfo}
+        />
+      ))}
+    </Stack>
+  );
+};
+
+export const TeamsList = () => {
+  return (
+    <Stack gap={4}>
+      <h1 style={{ margin: 0 }}>Teams</h1>
+      The Solus project is developed and maintained by the Solus Staff group. That body is made up of smaller teams of
```

Comment:
I'm confused by this. The Solus Staff group isn't made up of smaller teams?

Reply:
My understanding of the Solus organization was that *everyone* was a part of Solus Staff, which is then further broken up into smaller subteams. If that is not true, then we really need to figure this out because it is very confusing.